js/ui/calendar/ui.calendar.base_view.js

```diff
@@ -274,15 +274,16 @@ const BaseView = Widget.inherit({
             return;
         }
 
-        const value = this.option('value');
-        const selectedCell = this._getCellByDate(value);
-
-        if(this._selectedCell) {
-            this._selectedCell.removeClass(CALENDAR_SELECTED_DATE_CLASS);
+        if(this._selectedCells) {
```

Comment:
1. we can just write instead of this condition
2. please, make arrow function's formatting more readable
`this._selectedCells?.forEach(cell => { cell.removeClass(CALENDAR_SELECTED_DATE_CLASS); });`